Upgrade Jira integration to export issues by sprint (#1911)
port-labs/port-docs: docs/build-your-software-catalog/sync-data-to-catalog/project-management/jira/jira.md

deleteDependentEntities: true
resources:
  - kind: issue
    selector:
      // highlight-next-line
      query: .fields.customfield_11111[-1].name == "Sprint 35"  # Replace "Sprint 35" with the name of the current sprint
      jql: "statusCategory != Done"
    port:
      entity:
        mappings:
          identifier: .key
          title: .fields.summary
          blueprint: '"jiraIssue"'
          properties:
            url: (.self | split("/") | .[:3] | join("/")) + "/browse/" + .key
            status: .fields.status.name
            issueType: .fields.issuetype.name
            components: .fields.components
            assignee: .fields.assignee.emailAddress
            reporter: .fields.reporter.emailAddress
            creator: .fields.creator.emailAddress
            priority: .fields.priority.name
            // highlight-next-line
            sprint: .fields.customfield_11111 | sort_by(.id) | .[-1].name // ""
            created: .fields.created
            updated: .fields.updated
          relations:
            project: .fields.project.key
            parentIssue: .fields.parent.key
            subtasks: .fields.subtasks | map(.key)
```

</details>

> :::note Issues with blank Sprint values
> 
> It is to be noted that if the `createMissingRelatedEntities` is set to `true`, issues with blank sprint values will be created in Port thereby showing some issues with blank `Sprint` fields and empty or null properties. To avoid this, set `createMissingRelatedEntities` to `false`.
> :::

Reviewer: ```suggestion
:::info Issues with blank Sprint values
If the `createMissingRelatedEntities` is set to `true`, issues with blank `Sprint` values and some empty/null properties will be created in Port.  
To avoid this, set `createMissingRelatedEntities` to `false`.
:::
```
